app/services/jira_client/repository.rb

```diff
@@ -45,7 +45,7 @@ def sprints
 
     def create_request(issue_type)
       @request_params = {
-        jql: "project=#{@jira_board.jira_project_key} AND #{issue_type}",
+        jql: "project='#{@jira_board.jira_project_key}' AND #{issue_type}",
```

Comment:
[String interpolation only works with double quotes.](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html)

Reply:
In this case the interpolation is with double quotes but, inside single (or double) quotes are needed around the key string literal as part of the request params.
So, reading the documentation it may seem that it is better to escape` "...\"#{key}\"..."` ... or what about the [percent notation](https://blog.appsignal.com/2016/12/21/ruby-magic-escaping-in-ruby.html) `%(..."#{key}"...)` ? 

WDYT?

pd: i have tried all 3 options and they all worked.

Reply:
Oh, I see. Thanks for the clarification!